package/MDAnalysis/analysis/hole.py

```diff
@@ -356,6 +362,7 @@ def plot(self, **kwargs):
         ax.set_ylabel(r"HOLE radius $r$ ($\AA$)")
         if kw['label']:
             ax.legend(loc="best")
+        return ax
```

Comment:
One can do this in one line
```python
ax = kwargs.pop('ax', plt.gca())
```
at the cost that `plt.gca()` is always executed. Does this 1-line version have a disadvantage over the 3-line version?